src/v8/v8.cc

```diff
@@ -599,8 +603,8 @@ void V8::getModuleFunctionImpl(std::string_view function_name,
     SaveRestoreContext saved_context(context);
     auto trap = func->call(params, nullptr);
     if (trap) {
-      fail(FailState::RuntimeError, "Function: " + std::string(function_name) + " failed: " +
-                                        std::string(trap->message().get(), trap->message().size()));
+      auto message = this->get_fail_message(std::string(function_name), std::move(trap));
+      fail(FailState::RuntimeError, message);
```

Comment:
Is there any reason for the explicit assignment here instead of simply using:
```
fail(FailState::RuntimeError, this->get_fail_message(std::string(function_name), std::move(trap)));
```
?